lectures/L06-slides.tex

```diff
@@ -86,7 +86,7 @@
 
 	Synchronous Receive: the receiver is blocked until it receives a message.
 
-	Asynchronous Receive: the receiver is notified there is no message available and continues execution.
+	Asynchronous Receive: the receiver continues execution and is notified when there is a message available.
```

Comment:
I think this was better as it was. The idea we're trying to get with asynchronous receive is like when you go to your mailbox and find it is empty. You checked it, there's no mail available, so you continue with your day. You can check back later, if you want, but you don't get notified if there's mail available.